### PR TITLE
Fix allowlisted bin fields & add dates to gallery submissions

### PR DESCRIPTION
**Pull request by claynicholson**

### CODE REVIEW

File: pages/api/bin/gallery/posts.js [hunk 1]
```diff
@@ -7,18 +7,21 @@ const fetchPosts = async () => {
       baseID: 'appKjALSnOoA0EmPk',
       tableName: 'Main',
     });
-
+//process.env.AIRTABLE_API_KEY
```
Comment (maxwofford) [inline pages/api/bin/gallery/posts.js]: Still need this?

File: pages/bin/gallery.js [hunk 2]
```diff
@@ -7,31 +7,20 @@ import { useEffect, useRef, useState } from 'react';
...
-      setAllPosts(data.filter(post => post.Status === 'Accepted')); //Filter out rejected or under review as well as hidden posts
-    } catch (error) {
-      throw error;
-    } finally {
-      // Set loading to false when the async function is done
-      setLoading(false);
-    }
-  }
+function Gallery({ posts = [] }) {
+  console.log(posts)
```
Comment (maxwofford) [inline pages/bin/gallery.js]: Need this comment?

**Comment by lachlanjc** [comment]
#### 📝 Changed routes:


| [**/bin/gallery**](https://site-groo6nq6i.hackclub.dev/bin/gallery) |
|:-:|
|<a href="https://site-groo6nq6i.hackclub.dev/bin/gallery">Screenshot of /bin/gallery</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/uFdAqWr3WisSXt7MXHN-vKE0_zB3gGdQOE3iZ6LIV9NfB1QwTJt8w1zWellcaa7EqSerygn8bwhZw0HpywDinA.png">(view full size)</a> |



And 1 other route:
- [**/api/bin/gallery/posts**](https://site-groo6nq6i.hackclub.dev/api/bin/gallery/posts)



Commit c44533e762f8649c2bf74ec3d9b5492217bcebf4 (https://site-groo6nq6i.hackclub.dev).

**Comment by maxwofford** [comment]
This is approved for merge once you get those comments cleaned up!

**Pull request merged**